.vscode/settings.json

```diff
@@ -1,8 +1,3 @@
 {
-  "eslint.workingDirectories": ["tests"],
-  "workbench.colorCustomizations": {
-    "activityBar.background": "#561529",
-    "titleBar.activeBackground": "#781E3A",
-    "titleBar.activeForeground": "#FEFBFC"
-  }
+  "eslint.workingDirectories": ["tests"]
```

Comment:
What is going on here?

Reply:
vscode theme turned to red since someone changed the editor color in one of their commits. I reverted those changes since it's not necessary to overridde anyone'e vscode theme color for working on the repo